src/cata_utility.cpp

```diff
@@ -225,9 +225,46 @@ int bound_mod_to_vals( int val, int mod, int max, int min )
     return mod;
 }
 
-std::string weight_units()
+const char*  velocity_units( const units_type vel_units )
 {
-    return OPTIONS["USE_METRIC_WEIGHTS"].getValue() == "lbs" ? _("lbs") : _("kg");
+    if( OPTIONS["USE_METRIC_SPEEDS"].getValue() == "mph" ) {
+        return _( "mph" );
+    } else {
+        switch( vel_units ) {
+        case VU_VEHICLE:
+            return _( "km/h" );
+        case VU_WIND:
+            return _( "m/s" );
+        }
+    }
+}
```

Comment:
Sure. Alternatively, you can switch the if branches:
```C++
if( ... ) {
    switch(...) {
    }
}
return "mph";
````

But returning an error string is probably better.